src/code/z_kaleido_setup.c

```diff
@@ -84,7 +84,7 @@ void KaleidoSetup_Update(PlayState* play) {
             pauseCtx->mainState = PAUSE_MAIN_STATE_SWITCHING_PAGE;
 
             //! @bug REG collision
-            if (R_START_LABEL_DD(0) == 0) {
+            if (ZREG(48) == 0) {
```

Comment:
Can you add to the comment which regs this is confonded with

afaict R_START_LABEL_SCALE for NTSC and R_START_LABEL_DD(0) for PAL